src/Configuration/Programmer/Programmer.cpp

```diff
@@ -60,6 +61,7 @@ std::map<int, std::string> ErrorMessages = {
 
 void programmer_entry(const CFGCommon_ARG* cmdarg) {
   auto arg = std::static_pointer_cast<CFGArg_PROGRAMMER>(cmdarg->arg);
+  auto nonConstCmdArg = const_cast<CFGCommon_ARG*>(cmdarg);
```

Comment:
I think it is better to remove this `const_cast` and remove const from `cmdarg`. It makes code mode clear.